content/03.results.md

```diff
@@ -227,21 +227,21 @@ Interestingly, none of the hypermutant samples showed evidence of signature 3 (p
-| PT_JNEV57VK               | BS_QWM9BPDY               | 7316-3058 | Progressive       | Derived Cell Line | Nivolumab                                        | Lynch Syndrome                      | MSH2                        | 7.4  | DMG, H3 K28, TP53 loss             |
-| PT_JNEV57VK               | BS_P0QJ1QAH               | 7316-3058 | Progressive       | Solid Tissue      | Nivolumab                                        | Lynch Syndrome                      | MSH2                        | 6.3  | DMG, H3 K28, TP53 activated        |
-| PT_S0Q27J13               | BS_P3PF53V8               | 7316-2307 | Initial CNS Tumor | Solid Tissue      | Temozolomide, Irinotecan, Radiation              | None documented                     | None detected               | 15.5 | HGG, H3 wildtype, TP53 activated   |
-| PT_VTM2STE3               | BS_ERFMPQN3               | 7316-2189 | Progressive       | Derived Cell Line | Unknown                                          | Lynch Syndrome                      | None detected               | 5.7  | HGG, H3 wildtype, TP53 loss        |
-| PT_VTM2STE3               | BS_02YBZSBY               | 7316-2189 | Progressive       | Solid Tissue      | Unknown                                          | Lynch Syndrome                      | None detected               | 274.5 | HGG, H3 wildtype, TP53 activated   |
+| Kids First Participant ID | Kids First Biospecimen ID | CBTN ID   | Phase of therapy  | Composition       | Therapy post-biopsy                              | Cancer predisposition(s)       | Pathogenic germline variant             | TMB                | OpenPBTA molecular subtype         |
+|---------------------------|---------------------------|-----------|-------------------|-------------------|--------------------------------------------------|--------------------------------|-----------------------------------------|--------------------|------------------------------------|
+| PT_0SPKM4S8               | BS_VW4XN9Y7               | 7316-2640 | Initial CNS Tumor | Solid Tissue      | Temozolomide, CCNU, Radiation                    | None documented                | PMS2 p.S46I (LP)                        | 187.38765454966892 | HGG, H3 wildtype, TP53 activated   |
+| PT_3CHB9PK5               | BS_20TBZG09               | 7316-515  | Initial CNS Tumor | Solid Tissue      | Temozolomide, Irinotecan, Bevacizumab, Radiation | Other inherited conditions NOS | MSH6 NM_000179.2 c.3439-2A>G splice (P) | 307.0492167109247  | HGG, H3 wildtype, TP53 loss        |
+| PT_3CHB9PK5               | BS_8AY2GM4G               | 7316-2085 | Progressive       | Solid Tissue      | Unknown                                          | Lynch syndrome                 | MSH6 NM_000179.2 c.3439-2A>G splice (P) | 321.6079467820181  | HGG, H3 wildtype, TP53 loss        |
```

Comment:
Looking at this long variant description, I'm wondering if we might want to include ClinVar accessions IDs? For this example, it would be `VCV000089391` Alternatively, we could leave all the other the same and `MSH6 c.3439-2A>G (P)` should be sufficient for identification here (it yields a unique search result in ClinVar)

Reply:
I think this may work, but let me confer with Sharon who requested that we put the transcript ID in when showing cDNA changes. It looks like the current record is VCV000089391.27 and lists various transcript IDs, so may be ok.

Reply:
I don't think the portion after the decimal is required, as that will be updated as information is added to the DB, but will always refer to the same mutation.

Reply:
Hmm ok, yeah I guess I would need the actual clinvar annotation to go with the version. Would it be ok if I just did a line break if the issue is that it's too long?

Reply:
A line break would be fine. I'm probably more concerned about consistency and easy searching.